rfcs/0000-docstrings.md

```diff
@@ -0,0 +1,101 @@
+- Feature Name: docstrings
+- Start Date: 2020-06-27
+- RFC PR: [FuelLabs/sway-rfcs#0000](https://github.com/FuelLabs/sway-rfcs/pull/2)
+- Sway Issue: [FueLabs/sway#149](https://github.com/FuelLabs/Sway/issues/149)
+
+# Summary
+
+[summary]: #summary
+
+Sway will convert Rust-style docstrings (`///`-style comments) into a `#[doc("documentation string")]` annotation. This annotation can then be consumed by tooling and plugins to produce
+rendered documentation, a la `cargo doc` or `docs.rs`.
```

Comment:
These end lines make the text a bit harder to read and review. I personally like the `gq` command in Vim for consistency (even though it adds more end lines), but a single long line is also fine. Totally up to you though :)